### PR TITLE
*: disable telemetry by default

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)
Telemetry is disabled by default. This PR modified all documents that relate to telemetry.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: telemetry.md [hunk 1]
```diff
@@ -62,6 +68,10 @@ TIUP_CLUSTER_DEBUG=enable tiup cluster list
 
 ### TiSpark
 
+> **注意：**
+>
+> 自 TiSpark v3.3 起，默认关闭遥测功能，即 TiSpark 默认不收集使用情况信息，不将这些信息分享给 PingCAP 用于改善产品。
```
Comment (TomShawn) [inline telemetry.md:73]:
```suggestion
> TiSpark 自 v3.3 起，默认关闭遥测功能，即 TiSpark 默认不收集使用情况信息，不将这些信息分享给 PingCAP 用于改善产品。
```

File: telemetry.md [hunk 2]
```diff
@@ -6,7 +6,13 @@ aliases: ['/docs-cn/dev/telemetry/']
...
 # 遥测
 
-TiDB、TiUP 及 TiDB Dashboard 默认会收集使用情况信息，并将这些信息分享给 PingCAP 用于改善产品，例如，通过这些使用信息，PingCAP 可以了解常见的 TiDB 集群操作，从而确定新功能优先级。
+开启遥测后，TiDB、TiUP 及 TiDB Dashboard 会收集使用情况信息，并将这些信息分享给 PingCAP 用于改善产品，例如，通过这些使用信息，PingCAP 可以了解常见的 TiDB 集群操作，从而确定新功能优先级。
+
+> **注意：**
+>
+> 2023 年 2 月 20 日及之后发布的 [TiDB](/releases/release-timeline.md) 和 TiDB Dashboard 版本中，默认关闭遥测功能，即 TiDB 和 TiDB Dashboard 默认不收集使用情况信息，不将这些信息分享给 PingCAP 用于改善产品。
+>
+> TiUP 从 v1.11.3 版本起，默认关闭遥测功能，新部署的 TiUP 默认不再收集使用情况分享给 PingCAP。
```
Comment (TomShawn) [inline telemetry.md:15]:
```suggestion
> 在 v6.6.0 及之后发布的里程碑版本 (DMR) 和长期支持版本 (LTS) 中，TiDB 和 TiDB Dashboard 默认关闭遥测功能，即默认不再收集使用情况信息。如果升级前使用默认的遥测配置，则升级后遥测功能处于关闭状态。
>
> 此外，2022 年 2 月 20 日后，为 TiDB LTS 版本发布的补丁版本也默认关闭遥测功能，默认不再收集使用情况信息分享给 PingCAP。具体的版本可参考 [TiDB 版本发布时间线](/releases/release-timeline.md)。
>
> 从 v1.11.3 起，新部署的 TiUP 默认关闭遥测功能，即默认不再收集使用情况信息。如果从 v1.11.3 之前的 TiUP 版本升级至 v1.11.3 或更高 TiUP 版本，遥测保持升级前的开启或关闭状态。
```

**Review by zhangyangyu — APPROVED** [review]

File: telemetry.md [hunk 3]
```diff
@@ -6,7 +6,12 @@ aliases: ['/docs-cn/dev/telemetry/']
 
 # 遥测
 
-TiDB、TiUP 及 TiDB Dashboard 默认会收集使用情况信息，并将这些信息分享给 PingCAP 用于改善产品，例如，通过这些使用信息，PingCAP 可以了解常见的 TiDB 集群操作，从而确定新功能优先级。
+开启遥测后，TiDB、TiUP 及 TiDB Dashboard 会收集使用情况信息，并将这些信息分享给 PingCAP 用于改善产品，例如，通过这些使用信息，PingCAP 可以了解常见的 TiDB 集群操作，从而确定新功能优先级。
+
+> **注意：**
+>
+> - 自 2023 年 2 月 20 日起，新发布的 TiDB 和 TiDB Dashboard 版本（含 v6.6.0），默认关闭遥测功能，即默认不再收集使用情况信息分享给 PingCAP。如果升级至这些版本前使用默认的遥测配置，则升级后遥测功能处于关闭状态。具体的版本可参考 [TiDB 版本发布时间线](/releases/release-timeline.md)。
```
Comment (qqqdan) [inline telemetry.md:13]:
```suggestion
> - 自 2023 年 2 月 20 日起，新发布的 TiDB 和 TiDB Dashboard 版本（含 v6.6.0），默认关闭遥测功能，即默认不再收集使用情况信息分享给 PingCAP。如果集群在升级至这些版本前使用默认的遥测配置，则升级后遥测功能处于关闭状态。具体的版本可参考 [TiDB 版本发布时间线](/releases/release-timeline.md)。
```

**Review by qqqdan — APPROVED** [review]

File: telemetry.md [hunk 4]
```diff
@@ -75,12 +84,191 @@ TIUP_CLUSTER_DEBUG=enable tiup cluster list
 ```shell
 cat {spark.log} | grep Telemetry report | tail -n 1
 ```
+## 开启遥测功能
```
Comment (TomShawn) [inline telemetry.md:87]:
```suggestion

## 开启遥测功能
```

File: telemetry.md [hunk 5]
```diff
@@ -76,11 +85,191 @@ TIUP_CLUSTER_DEBUG=enable tiup cluster list
 cat {spark.log} | grep Telemetry report | tail -n 1
 ```
 
+## 开启遥测功能
+
+### 开启 TiDB 遥测
+
+#### 步骤 1：修改配置文件中的遥测配置
+
+在已有 TiDB 集群上开启遥测功能，需要为集群设置 [`enable-telemetry = true`](/tidb-configuration-file.md#enable-telemetry-从-v402-版本开始引入) ，但需要重启集群后才能生效。
```
Comment (TomShawn) [inline telemetry.md:94]:
```suggestion
在已有 TiDB 集群上开启遥测功能，需要为集群设置 [`enable-telemetry = true`](/tidb-configuration-file.md#enable-telemetry-从-v402-版本开始引入)，但需要重启集群后才能生效。
```

File: telemetry.md [hunk 6]
```diff
@@ -76,11 +85,191 @@ TIUP_CLUSTER_DEBUG=enable tiup cluster list
 cat {spark.log} | grep Telemetry report | tail -n 1
 ```
 
+## 开启遥测功能
+
+### 开启 TiDB 遥测
+
+#### 步骤 1：修改配置文件中的遥测配置
```
Comment (TomShawn) [inline telemetry.md:92]:
```suggestion
#### 第 1 步：修改配置文件中的遥测配置
```

File: telemetry.md [hunk 7]
```diff
@@ -76,11 +85,191 @@ TIUP_CLUSTER_DEBUG=enable tiup cluster list
...
+
+详情参见[在标准 Kubernetes 上部署 TiDB 集群](https://docs.pingcap.com/zh/tidb-in-kubernetes/stable/deploy-on-general-kubernetes)。
+
+> **注意：**
+>
+> 该配置需使用 TiDB Operator v1.1.3 或更高版本才能生效。
+
+</details>
+
+#### 步骤 2：修改系统全局变量的遥测配置
```
Comment (TomShawn) [inline telemetry.md:166]:
```suggestion
#### 第 2 步：修改系统全局变量的遥测配置
```

File: telemetry.md [hunk 8]
```diff
@@ -76,11 +85,191 @@ TIUP_CLUSTER_DEBUG=enable tiup cluster list
...
+
+在步骤 1 完成后，还需要修改系统全局变量 [`tidb_enable_telemetry`](/system-variables.md#tidb_enable_telemetry-从-v402-版本开始引入) ，才能打开 TiDB 遥测功能：
```
Comment (TomShawn) [inline telemetry.md:168]:
```suggestion
在步骤 1 完成后，还需要修改系统全局变量 [`tidb_enable_telemetry`](/system-variables.md#tidb_enable_telemetry-从-v402-版本开始引入)，才能打开 TiDB 遥测功能：
```

File: telemetry.md [hunk 9]
```diff
@@ -76,11 +85,191 @@ TIUP_CLUSTER_DEBUG=enable tiup cluster list
...
+
+{{< copyable "shell-regular" >}}
+
+```shell
+tiup telemetry enable
+```
+
+### 开启 TiSpark 遥测
+
+可以通过在 Spark 配置文件设置 `spark.tispark.telemetry.enable = true` 来禁用 TiSpark 的遥测功能。
```
Comment (TomShawn) [inline telemetry.md:266]:
```suggestion
可以通过在 Spark 配置文件设置 `spark.tispark.telemetry.enable = true` 来开启 TiSpark 的遥测功能。
```

File: tiup/tiup-overview.md [hunk 10]
```diff
@@ -33,7 +33,7 @@ tiup --version
 
 > **注意：**
 >
-> TiUP 默认会收集使用情况信息，并将这些信息分享给 PingCAP 用于改善产品。若要了解所收集的信息详情及如何禁用该行为，请参见[遥测](/telemetry.md)。
+> 对于 v1.11.3 及以上版本的 TiUP，默认不会收集使用情况信息分享给 PingCAP。若要了解所收集的信息详情及如何关闭分享行为，请参见[遥测](/telemetry.md)。
```
Comment (ran-huang) [inline tiup/tiup-overview.md:36]: 这里是不是像前面几个文档里一样，整个 note 删掉更好

**Review by ran-huang — APPROVED** [review]
Rest LGTM

File: tiup/tiup-overview.md (hunk 10 above)
Comment (TomShawn) [reply tiup/tiup-overview.md:36]: TiUP 是独立版本，而这个文档是跟着 TiDB 文档版本走的。高版本的 TiDB 可以使用低版本 TiUP 的，所以这里不能直接删除。

Comment (ran-huang) [reply tiup/tiup-overview.md:36]: LGTM

**Pull request merged**